lib/beaker/host.rb

```diff
@@ -499,6 +501,7 @@ def do_rsync_to from_path, to_path, opts = {}
     'freebsd',
     'windows',
     'pswindows',
+    'eos',
```

Comment:
ok.  I think the current way is one of the more deterministic routes, so I'm down with just keeping it for now.